armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -54,6 +53,9 @@ class FuelHandler:
     with the ``fuelHandler`` setting. In that file, subclass this object.
     """
 
+    # Import functions
+    from armi.physics.fuelCycle import translationFunctions
```

Comment:
Shouldn't this be at the top of the file?

Unless you tried that and it causes some sort of cyclic import problem you're trying to avoid?

Reply:
Yes. This should be. It is now moved to top of file.